arch/risc-v/src/common/riscv_mmu.h

```diff
@@ -158,7 +158,7 @@ extern uintptr_t g_kernel_pgt_pbase;
  * Public Function Prototypes
  ****************************************************************************/
 
-void weak_function mmu_flush_cache(void);
+void weak_function mmu_flush_cache(uintptr_t);
```

Comment:
It is better to follow api defined in include/nuttx/cache.h, all other subsystem(e.g. OpenAMP) already integrate with this api set.

Reply:
@xiaoxiang781216 thanks for menioning that cache API.
 
Actually this hook is used for special cases like invalidating cached page table memory, as those hacks are treated as chip errata in Linux, so guessing normally this operation is not needed if the chip correctly handles it when `satp` register is updated. do you think if it is still worth to add it as general API?

Reply:
OK